src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/DirectoryControl.cs

```diff
@@ -50,7 +50,6 @@ internal static class UtilityHandle
         public static ConnectionHandle GetHandle() => s_handle;
     }
 
-    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
```

Comment:
Well, the class is not marshalled anymore. If it's for some kind of API/docs compatibility then I can put it back but it shouldn't be necessary. What do you think?

Reply:
The layout attribute came this way from .NET Framework.  My feeling is that we should leave it unless we have an active reason to remove it.  (Though it looks like it's missing from the ref.cs file: https://github.com/dotnet/runtime/blob/bb1afab3dd68a271c0f43cb8696183f8121c5a8c/src/libraries/System.DirectoryServices.Protocols/ref/System.DirectoryServices.Protocols.cs#L659-L661; so removing it might be a reasonable thing to do as a followup... like after checking if System.DirectoryServices.Protocols' ref.cs file is authoritative or just informative... or if the attribute matters to compilation (when on a reference type), etc)

Reply:
Done, thanks!